src/intl.store.js

```diff
@@ -1,6 +1,7 @@
 import { Store } from "lux.js";
 import IntlMessageFormat from "intl-messageformat";
 import { merge, cloneDeep } from "lodash";
+import window from "window";
```

Comment:
This isn't an actual module, so we cannot use it here. Just remove this line, and in the tests do something like this:

```js
// in beforeEach
numberStub = sinon.stub( window.Intl, "NumberFormat" )

// in afterEach
numberStub.restore();
```